### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by cristicirdei**
Restructured routes and models. Added preliminary version of _email routes_.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const server = new Server();
 
 require("dotenv").config();
 database.bootstrap();
-server.bootstrap();
+server.bootstrap();
```
Comment (sonicristea) [inline src/index.js]: Remove the additional space, so that no unnecessary change is tracked ;)

File: src/bootstrap/server.bootstrap.js [hunk 2]
```diff
@@ -38,6 +41,9 @@ export class Server {
     this.app.use("/health", healthRouter);
     this.app.use("/events", eventRouter);
     this.app.use("/user", userRouter);
+    this.app.use("/my", myRouter);
```
Comment (sonicristea) [inline src/bootstrap/server.bootstrap.js:44]: I see you have here a separate controller for calling user specific routes (in this case, your own, right?).
Wouldn't it be an idea to call the routes in this controller in their specific one and add the user id as a parameter?

e.g the `getMyEvents` route can be called in the `events` controller using the simple `get all events` method, which now could have the userid sent as parameter?

File: src/models/invite.js [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
     }
 });
 
 // Create model
 const Invite = mongoose.model("invite", InviteSchema);
 
 //Export model
-module.exports = Invite;
\ No newline at end of file
+module.exports = {Invite, InviteSchema};
```
Comment (sonicristea) [inline src/models/invite.js]: Here please add a new line at the end of the file

File: src/bootstrap/server.bootstrap.js (hunk 2 above)
Comment (AndreiGutaIBM) [inline src/bootstrap/server.bootstrap.js:44]: Why did we add this? What functionality does it cover?

Comment (cristicirdei) [reply src/bootstrap/server.bootstrap.js:44]: Yes, I can see how it could be better this way. I'll make the changes. Thanks!

Comment (cristicirdei) [reply src/bootstrap/server.bootstrap.js:44]: It was intended to _handle_ requests for a specific user (i.e. the owner of certain events). But, as noted by @sonia-cristea we could solve this _problem_ in a simpler way.
